src/main/modules/helmet/index.ts

```diff
@@ -27,7 +27,7 @@ export class Helmet {
 
     this.setContentSecurityPolicy(app);
     this.setReferrerPolicy(app, this.config.referrerPolicy);
-    this.setHttpPublicKeyPinning(app, this.config.hpkp);
+    // this.setHttpPublicKeyPinning(app, this.config.hpkp);
   }
```

Comment:
Commented out until we have the proper key

Reply:
OK